### PR TITLE
`<replay-web-page>` 1.7.0 and other minor playback page edits

### PR DESCRIPTION
**Pull request by matteocargnelutti**
Implements #3182 and proposes "something" for #3140 

---

### Proposed replacement for _"Welcome"_ when landing on `replay.perma.cc`:

Screen Shot 2022-10-05 at 3 17 12 PM

---

### Other considerations
Tightened sandboxing rules for parent `<iframe>` on playback page. Will need to test extensively on staging to make sure it doesn't interfere with playback.

### CODE REVIEW

File: perma_web/replay/templates/iframe.html [hunk 1]
```diff
@@ -21,31 +33,59 @@
...
+      color: #333333;
+      padding: 1rem;
+    }
+
+    #no-playback div {
+      text-align: center;
+    }
+
+    #no-playback p {
+      font-size: clamp(20px, 3vmax, 36px);
```
Comment (lizadaly) [inline perma_web/replay/templates/iframe.html:72]: I just started using `clamp()` and I love it.

**Review by lizadaly — APPROVED** [review]

**Review by rebeccacremona — APPROVED** [review]
Huzzah!

One thought: on reading, "Tip: You may use this page to clear your playback cache," I might ask, "Cool, how?" Perhaps something like, "Tip: visiting this page clears your playback cache" would be clearer?

File: perma_web/replay/templates/iframe.html (hunk 1 above)
Comment (rebeccacremona) [reply perma_web/replay/templates/iframe.html:72]: Ooh, I have not yet learned about `clamp()`.

**Comment by matteocargnelutti** [comment]
Thank you both, @lizadaly and @rebeccacremona !

---

@rebeccacremona: Yes indeed! Edited as suggested 😄 .

**Pull request merged**